filters/openpolicyagent/openpolicyagent.go

```diff
@@ -599,7 +599,7 @@ func (opa *OpenPolicyAgentInstance) startSpanFromContextWithTracer(tr opentracin
 }
 
 func (opa *OpenPolicyAgentInstance) StartSpanFromFilterContext(fc filters.FilterContext) (opentracing.Span, context.Context) {
-	return opa.startSpanFromContextWithTracer(fc.Tracer(), fc.ParentSpan(), fc.Request().Context())
+	return opa.StartSpanFromContext(fc.Request().Context())
```

Comment:
@mjungsbluth BTW I am not sure why OPA needs a whole set of opentracing helpers.

I think it should get tracer from its options and for synchronous filter requests it should get proxy span (and its tracer if needed) via `opentracing.SpanFromContext(ctx.Request().Context())`

Reply:
@AlexanderYastrebov It should be fine the way you outlined. It was probably overly defensive to include the tracer for cases where the context does not already contain a span (which actually don't provide much value for the synchronous cases).